binding.gyp

```diff
@@ -1,63 +1,70 @@
 {
-  'targets': [
+  "targets": [
     {
-      'target_name': 'blst_ts_addon',
-      'sources': [
-        'blst/src/server.c',
-        'blst/build/assembly.S',
-        'src/addon.cc',
-        'src/secret_key.cc',
-        'src/public_key.cc',
-        'src/signature.cc',
-        'src/functions.cc',
+      "target_name": "blst_ts_addon",
+      "sources": [
+        "blst/src/server.c",
+        "src/addon.cc",
+        "src/secret_key.cc",
+        "src/public_key.cc",
+        "src/signature.cc",
+        "src/functions.cc",
       ],
-      'include_dirs': [
-        'blst/bindings',
-        "<!@(node -p \"require('node-addon-api').include_dir\")",
+      "include_dirs": [
+        "blst/bindings",
+        "<!@(node -p \"require('node-addon-api').include\")",
```

Comment:
This is a known issue with how windows builds native modules.  Need to use the `included export instead of `include_dir` for it to be fully cross-compatible.